tasks/collect-pyxis-params/README.md

```diff
@@ -0,0 +1,11 @@
+# collect-pyxis-params
+
+Tekton task that collects pyxis configuration options from the data file. The task looks at the data file
+in the workspace to extract the `server` and `secret` keys for Pyxis. These are both emitted as task results
+for downstream tasks to use.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| dataPath | Path to the JSON string of the merged data containing the pyxis configuration options to use | Yes | data.json |
```

Comment:
Nit: It's a JSON file, not a string. So maybe something like this?

"Path to the merged data JSON file generated by collect-data task and containing the pyxis configuration options to use"

Reply:
nice catch